scalardb/src/scalardb/db/cluster.clj

```diff
@@ -39,7 +39,10 @@
                "scalar.db.storage=jdbc"
                "scalar.db.contact_points=jdbc:postgresql://postgresql-scalardb-cluster.default.svc.cluster.local:5432/postgres"
                "scalar.db.username=postgres"
-               "scalar.db.password=postgres"])
+               "scalar.db.password=postgres"
+               ""
+               ;; Set to true to include transaction metadata in the records
+               "scalar.db.consensus_commit.include_metadata.enabled=true"])
```

Comment:
I think this change also affects the cluster tests, so I added the configuration here as well.

Note that I didn't added support for the read-committed isolation level and the one-phase commit optimization in the cluster tests.